apps/backend/src/deposits/cash-deposit.service.ts

```diff
@@ -133,11 +134,11 @@ export class CashDepositService {
   async findCashDepositExceptions(
     date: string,
     program: Ministries,
-    pt_location_id: number
+    location: LocationEntity
   ): Promise<CashDepositEntity[]> {
     return await this.cashDepositRepo.find({
       where: {
-        pt_location_id,
+        pt_location_id: In(location.banks.map((bank) => bank.pt_location_id)),
```

Comment:
TODO: update cash deposit/pos deposit to find by the location (no need to destructure by the ids)